.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,33 @@
+name: reviewdog
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  reviewdog:
+    name: 🐶 Reviewdog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: reviewdog/action-setup@v1
+        with:
+          reviewdog_version: latest
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: "1.18"
```

Comment:
this will require linter config and as we talked in py repo, better to run with old version because SDK can change and newer versions can bring new suggestions but we might not able to those changes and then need to silence them (chore work)

Reply:
✅  set it to 1.17, assuming there are no more updates for 17 :)